### PR TITLE
Manual Respawn + Misc. Changes

### PR DESCRIPTION
**Pull request by EasterLee**

### CODE REVIEW

**Comment by EasterLee** [comment]
need help wildcarding setpawn sig for linux

File: src/zombiereborn.cpp [hunk 1]
```diff
@@ -515,24 +540,28 @@ void ZR_CheckWinConditions(bool bCheckCT)
...
-		{
-			pTeam->m_iScore = pTeam->m_iScore() + 1;
-			//Msg("%d, %d\n", pTeam->m_iTeamNum(), pTeam->m_iScore());
-			return;
-		}
-	}
+	if (iTeamNum == CS_TEAM_CT)
+		g_hTeamCT->m_iScore = g_hTeamCT->m_iScore() + 1;
+	else if (iTeamNum == CS_TEAM_T)
+		g_hTeamT->m_iScore = g_hTeamT->m_iScore() + 1;
```
Comment (Vauff) [inline src/zombiereborn.cpp:564]: You should probably still verify these handles are valid, and log an error/return if not. To save us from debugging crashes later on if we somehow fail to find `cs_team_manager` entities at map start.

**Comment by Vauff** [comment]
> need help wildcarding setpawn sig for linux

Doesn't look like it needs anything wildcarded. Don't think that one wildcard in the Windows sig is even needed.

File: cfg/cs2fixes/cs2fixes.cfg [hunk 2]
```diff
@@ -39,4 +39,6 @@ zr_infect_spawn_type			1		//Type of Mother Zombies Spawn [0 = MZ spawn where the
 zr_infect_spawn_time_min		15		//Minimum time in which Mother Zombies should be picked, after round start
 zr_infect_spawn_time_max		15		//Maximum time in which Mother Zombies should be picked, after round start
 zr_infect_spawn_mz_ratio		7		//Ratio of all Players to Mother Zombies to be spawned at round start
-zr_infect_spawn_mz_min_count	2		//Minimum amount of Mother Zombies to be spawned at round start
\ No newline at end of file
+zr_infect_spawn_mz_min_count	2		//Minimum amount of Mother Zombies to be spawned at round start
+zr_respawn_delay				5.0		//Time before a zombie is respawned
+zr_default_winner_team			1		//Which team wins when time ran out, use 1 for draw
```
Comment (Vauff) [inline cfg/cs2fixes/cs2fixes.cfg:44]: I would do the description like `zr_infect_spawn_type` and show all the values. Most people won't know team numbers for T/CT.

**Review by Vauff — CHANGES_REQUESTED** [review]

**Review by Vauff — APPROVED** [review]

**Pull request merged**